client/domain/FeedbackParagraphObjectFactory.js

```diff
@@ -140,6 +140,20 @@ tie.factory('FeedbackParagraphObjectFactory', [
       return new FeedbackParagraph(PARAGRAPH_TYPE_SYNTAX_ERROR, error);
     };
 
+    /**
+     * Returns a FeedbackParagraph created from a json object
+     */
+    FeedbackParagraph.createFromJson = function(json) {
+      if (json._type === "text") {
```

Comment:
Oops. I can change this (and the other occurrences) to the new constants that were added in #360 (it hasn't been merged yet I think).